docs/plugins/graphback-schema.md

```diff
@@ -35,16 +41,23 @@ npm install @graphback/codegen-schema
   outputFileName?: string
 ```
 
-Example YML: 
-```yml
- plugins:
-  graphback-schema:
-    format: 'graphql'
-    outputPath: ./server/src/schema
+
+### Usage
+
+```ts
+  const schemaPlugin = new SchemaCRUDPlugin({
+    format: 'graphql',
+    outputPath: './src/schema'
+  });
+
+  buildGraphbackAPI(modelDefs, {
```

Comment:
```suggestion
 const {schema, typeDefs, resolvers, contextCreator} = buildGraphbackAPI(modelDefs, {
```

To show the results of the above call?

Reply:
I can add it in, but did not bother as assumed they have this knowledge already. Good to be explicit though!